test/parser_test.cpp

```diff
@@ -0,0 +1,21 @@
+#include "yaml-cpp/parser.h"
+#include "mock_event_handler.h"
+#include "gtest/gtest.h"
+
+using YAML::Parser;
+using YAML::MockEventHandler;
+using ::testing::StrictMock;
+
+TEST(ParserTest, Empty) {
+    Parser parser;
+
+    EXPECT_FALSE(parser);
+
+    StrictMock<MockEventHandler> handler;
+    parser.HandleNextDocument(handler);
+    
+    std::ostringstream oss;
```

Comment:
Remove this, as discussed?

Reply:
@jbeder sorry, did wrong change, now everything looks as discussed